docs/wiki/[Examples]-Deploy-Custom-Management-Landing-Zone.md

```diff
@@ -0,0 +1,131 @@
+## Overview
+
+This page describes how to deploy Enterprise-scale management resources custom configuration, including guidance on how to apply Log Analytics and Azure Security Center preferences.
+In this example, we take a default configuration and make the following code changes:
+
+### Enable and configure management resources (Level 200)
+- Set and enable the `configure_management_resources`  variable
+- Add parameters for enabling/disabling management resources
+- Add parameters to set location and resource tags
+- Create custom configurations for Log Analytics and ASC
+
+
+The module allows for further configuration of log analytics and azure security center by setting the `configure_management_resources` variable.
+This configuration requires three mandatory variables:
+     location: where the resource group will be deployed
+     advanced:  [tbd]
+     tags: add any specific tag that the resources permit
+
+For this configuration you must pass in both log analytics and security center. If you want to disable
+either or both of these resources, set the enabled flag to *false*.
+
+```hcl
+    settings = {
+      log_analytics = {
+        enabled = false
+     }
+```
+If you've already deployed management resources, this will allow you to enable or disable specific parameters.
+Every parameter set to true will create new principal id's that will force replacement in place of the existing
+roles and policies to reassign them to the log analytics and sentinel resources specified. Likewise, if set to false, existing configurations will be removed.  
+
+If location is not specified, the resources will default to *eastus*
+```hcl
+    terraform {
+      required_providers {
+        azurerm = {
+            source = "hashicorp/azurerm"
+            version = ">=2.46.1"
+        }
+   
+     }
+    }
+
+    provider "azurerm"{
+    features {}
+    }
+
+    #Pull current Tenant ID from connection settings and store to data source
+    
+    data "azurerm_client_config" current {}
+
+    module "enterprise_scale" {
+      source = "Azure/caf-enterprise-scale/azurerm"
+      version = "0.3.1"
+
+    root_parent_id = data.azurerm_client_config.current.tenant_id
+    root_id = "contoso" 
+    root_name = "Contoso"
+    deploy_management_resources = "true"
+    subscription_id_management = "<insert_subscription_id>" 
```

Comment:
In this example, as we have set `deploy_management_resources = "true"`, we should be using the Management Subscription as the subscription set in the provider configuration (this is necessary to deploy resources into the correct Subscription).

As such, you can define `subscription_id_management = data.azurerm_client_config.current.subscription_id` which makes it possible for a customer to simply copy this code and it will work.

Reply:
Updated